guidelines/internal-guidelines/new-repository-settings.md

```diff
@@ -0,0 +1,93 @@
+
+## Use the repository template
+
+After you create a new repository, copy the basic repository structure from the [`template`](./repository-template/template) folder that is available in the `community` repository. It contains such files as the obligatory Apache license, the `CODEOWNERS` file that governs the review and approval flow in the repository, and the Stale Bot that handles inactive issues.
+
+> **TIP:** Read this [`README.md`](./repository-template/README.md) document to learn more about each file and document within the `template` folder.
+
+After copying the template documents and files, modify these according to the instructions they contain:
+- [`CODEOWNERS`](./repository-template/template/CODEOWNERS) to define who is responsible for the review and approval of specific repository parts
+- [`README.md`](./repository-template/template/README.md) to describe the repository and explain how to use and develop it
+
+## Adjust repository options
+
+Under the repository name, click **Settings**. The **Options** view opens as the default one in the left menu.
+
+1. Scroll down to the **Features** section and clear these options:
+- Wikis
+- Restrict editing to users in teams with push access only
+- Projects
+
+![Features](../../assets/features.png)
+
+2. Go to the **Merge button** section and clear these options:
+- Allow merge commits
+- Allow rebase merging
+
+![Merge button](../../assets/merge-button.png)
+
+Leave only the **Allow squash merging** option selected. This option combines all commits into one before merging the changes into the `master` branch.
+
+## Set branch protection rules
+
+Define branch protection rules that include enforcing obligatory review and approval of pull requests (PRs), and define which Prow jobs need to pass successfully before merging PR changes into the `master` branch.
+
+To see these settings, go to **Branches** in the left menu, under repository **Settings**:
+
+![Branch protection rules](../../assets/branch-protection-rules.png)
+
+In Kyma, these rules are defined in the Prow [`config.yaml`](https://github.com/kyma-project/test-infra/blob/master/prow/config.yaml#L368) file and handled by a Prow component called [Branch Protector](https://github.com/kyma-project/test-infra/blob/master/docs/prow/prow-architecture.md#branch-protector).
+
+If you add a new repository in:
+- `kyma-project`, you do not need to add a new entry to the Prow `config.yaml` file as the branch protection is already defined for [all repositories](https://github.com/kyma-project/test-infra/blob/master/prow/config.yaml#L380) within this organization. The only exception is if you want to specify additional rules that are not handled by Prow.
+- `kyma-incubator`, add a new repository entry to the Prow `config.yaml` file, under **branch-protection.kyma-incubator.repos**. [See](https://github.com/kyma-project/test-infra/blob/master/prow/config.yaml#L422) an example of such an entry for the `varkes` repository.
```

Comment:
I would rather use `marketplaces` repository as it has protection enabled on all branches